src/app_service/service/node_configuration/service.nim

```diff
@@ -182,6 +183,20 @@ method setFleet*(self: Service, fleet: string): bool =
     else:
       discard
 
+  if fleetType == Fleet.StatusTest:
```

Comment:
Once https://github.com/status-im/infra-nim-waku/issues/42 is implemented, we need to remove these hardcoded values from here.